brave/browser-laptop
# cleanup about page handling and fix missing document.l10n

fixes #6012

Test Plan:
For each about page:
1. Load the page
2. Ensure that the page and favicon load correctly
